### DISCUSSION TITLE
Unity editor fix

### DISCUSSION THREAD

**Discussion by Ciubix8513** [yshui/picom#1038] [Q&A]
Hi, I'm not sure if this is the right place to ask but I recently switched to i3 + picom from gnome and I'm experiencing an issue with the Unity engine editor.

The  editor ui isn't being redrawn when switching workplaces, and is only redrawn when moving the mouse over it. After a quick google search if found [this issue](https://issuetracker.unity3d.com/issues/linux-editor-only-redraws-panels-on-when-theyre-being-moused-over-when-no-compositor-is-present) over on unity bug tracker. 

Is there something I need to enable for it to work properly?

**Comment by xoorath** [comment]
I'm finding the same thing here 2 years later. I just wanted to add that I'm also seeing this if I kill the picom process, though.
